webui/apps/eidolon-ui2/src/components/UsageIndicator/UsageIndicator.tsx

```diff
@@ -15,7 +15,7 @@ export function UsageIndicator({...restOfProps}) {
   const {data: session} = useSession()
   useEffect(() => {
     if (session?.user?.id) {
-      usageForSession(session.user.id, eidolonContext.usageSeed).then((data) => {
+      usageForSession(session.user.id).then((data) => {
         setUsageData(data)
       })
     }
```

Comment:
deps (line 22) is how we should be driving recalculation. When it goes to the server, always recalculate, but if the seed remains the same there is no need.